src/platform/user/tests/authentication/downtime.unit.spec.js

```diff
@@ -2,40 +2,18 @@
 
 import { render } from '@testing-library/react';
 import { expect } from 'chai';
-import { SERVICE_PROVIDERS } from '../../authentication/constants';
 import * as downtimeUtils from '../../authentication/downtime';
 
 describe('generateCSPBanner', () => {
-  it('should return a specific banner message when csp is "idme"', () => {
-    const result = downtimeUtils.generateCSPBanner({
-      csp: 'idme',
-    });
-    expect(result).to.deep.equal({
-      headline: `You may have trouble signing in with some of your accounts`,
-      status: 'warning',
-      message: `We’re sorry. We’re working to fix some problems with ID.me. If you’d like to sign in to VA.gov with your ID.me, DS Logon, or My HealtheVet accounts, please check back later.`,
-    });
-  });
-
   // Array of CSPs to test
-  const csps = ['logingov', 'mhv', 'dslogon'];
+  const csps = ['logingov', 'idme', 'mhv', 'dslogon'];
```

Comment:
If we're going to include all the CSPs anyway. Just import the `SERVICE_PROVIDERS` use that.